packages/bundles/src/all-imports.js

```diff
@@ -0,0 +1,90 @@
+/* Core */
+import '@vaadin/accordion';
+import '@vaadin/app-layout';
+import '@vaadin/app-layout/vaadin-drawer-toggle.js';
+import '@vaadin/avatar';
+import '@vaadin/avatar-group';
+import '@vaadin/button';
+import '@vaadin/checkbox';
+import '@vaadin/checkbox-group';
+import '@vaadin/combo-box';
+import '@vaadin/context-menu';
+import '@vaadin/custom-field';
+import '@vaadin/date-picker';
+import '@vaadin/date-time-picker';
+import '@vaadin/details';
+import '@vaadin/dialog';
+import '@vaadin/email-field';
+import '@vaadin/form-layout';
+import '@vaadin/form-layout/vaadin-form-item.js';
+import '@vaadin/grid';
+import '@vaadin/grid/vaadin-grid-column-group.js';
+import '@vaadin/grid/vaadin-grid-column.js';
+import '@vaadin/grid/vaadin-grid-filter.js';
+import '@vaadin/grid/vaadin-grid-selection-column.js';
+import '@vaadin/grid/vaadin-grid-sorter.js';
+import '@vaadin/grid/vaadin-grid-tree-toggle.js';
+import '@vaadin/horizontal-layout';
+import '@vaadin/icon';
+import '@vaadin/icons';
```

Comment:
If we want to use deprecated `iron-icon`, we also need to also import `@vaadin/icons/iconset.js` which contains the iconset in a format that `iron-icon` supports - the default one only supports `vaadin-icon`.